control/autonomous_emergency_braking/src/node.cpp

```diff
@@ -452,8 +457,11 @@ void AEB::generateEgoPath(
     geometry_msgs::msg::Pose map_pose;
     tf2::doTransform(predicted_traj.points.at(i).pose, map_pose, transform_stamped);
     path.at(i) = map_pose;
-  }
 
+    if (i * mpc_prediction_time_interval_ > mpc_prediction_time_horizon_) {
```

Comment:
@1222-takeshi Since `mpc_prediction_time_interval_` is defined in the MPC, it should be calculated from predicted_trajectory itself.